examples/pylint/E0101_return_in_init.py

```diff
@@ -0,0 +1,24 @@
+"""pylint: return in init
+
+"""
+
+
+class Animal:
+    """An animal in the zoo.
+    """
+    def __init__(self, name):
+        self.name = name
+        return True
+
+    def is_animal(self):
+        """
+        @rtype: bool
+        """
+        return isinstance(self.name, str)
+
+    def getter(self):
```

Comment:
If the class doesn't have at least three public methods pylint raises another error for too few public methods. What should I do in this case?